Code/GraphMol/MolDraw2D/catch_main.cpp

```diff
@@ -0,0 +1,12 @@
+//
+//  Copyright (C) 2020 Greg Landrum
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do
```

Comment:
I thought that the main was in catch_qt.cpp?

Reply:
catch_main.cpp is currently used for `moldraw2DTestCatch`